src/flows/components/panel/FlowPanel.tsx

```diff
@@ -183,7 +206,7 @@ const FlowPanel: FC<Props> = ({
             <Handle
               dragRef={handleRef}
               onStartDrag={handleMouseDown}
-              dragging={isDragging}
+              dragging={isDragging == 2}
```

Comment:
```suggestion
              dragging={isDragging === 2}
```

<img src="https://media.giphy.com/media/f6pOe5e8ShRhS/giphy.gif?cid=ecf05e47nh51xb8l9kmafga8d8m9phuuxvvnvkwxbwrdjbx1&rid=giphy.gif&ct=g"  />